problem_2/spec/even_fibonacci_numbers_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'even_fibonacci_numbers'
```

Comment:
I would encourage you to have your first tests be even smaller. It'll help drive your code better. 
```
it 'returns sum of even fibonacci numbers under 3'
it 'returns sum of even fibonacci numbers under 15'
it 'returns sum of even fibonacci numbers under 100'
```